### ISSUE TITLE
Module Not found error - Angular 

### ISSUE THREAD

**Issue by sree332** [stackblitz/core#254]
I was able to install a npm module (material design bootstrap) but when I include it in the app.module.ts file I see an error - "Module Not Found "

**Comment by EricSimons** [comment]
@sree332 can you link the stackblitz project here?

**Comment by sree332** [comment]
https://stackblitz.com/edit/angular-m2mefa

**Comment by EricSimons** [comment]
@sree332 it looks like this is actually a problem with the `angular-bootstrap-md` package itself. Specifically, the [package.json](https://unpkg.com/angular-bootstrap-md@5.1.2/package.json) doesn't have the required `main` (CJS version of package), `module` (ESM version of package), or `typings` (TS type definition) fields. Instead, they only have an index.ts file of the raw TypeScript source files which is not how you're supposed to package & distribute these libs on NPM.

Do you know the folks who make this package by chance?

**Comment by sree332** [comment]
The package is maintained by https://mdbootstrap.com/material-design-for-bootstrap/ ,  I am using the free version in my projects for now and they have few setup instructions after downloading the package.

**Comment by mcristiano** [comment]
Hi, do you get a solution for this problem?

**Comment by sree332** [comment]
Hi,

No I did not find a solution so I just went back to cloud9 which seems much
stable at this point.

Thanks
Sree

**Comment by EricSimons** [comment]
@gmurray81 we're working on ensuring malformed npm packages can still work inside of stackblitz. However, it's worth noting that these sorts of packages will also fail out on other build systems (browserify, systemjs, etc), hence why it's best practice to properly create multiple bundles & fields to ensure all users can consume them without issues.

**Comment by gmurray81** [comment]
In our case, systemJS/browserify are de-prioritized use cases with some tangled hoops to jump through to make things right, so have been deferred for now. Which is why we don't even pretend that "main" is going to get you anywhere. Will hopefully change at some point. But in the short term, we care most about the AngularCLI primary scenarios.

**Comment by kalew8** [comment]
Any update on this? It's stopping me from being able to bring in bigger past projects into Stackblitz to play with since I don't want to refactor for this use case.

**Comment by EricSimons** [comment]
Yup! We’re reworking our bundler & Turbo npm client right now and this is one of the things that will be fixed up. Target for prod is by end of June 🙏 Will circle back once it’s online!

**Comment by gmurray81** [comment]
Any updates on this? I notice its still not working @EricSimons
